source/view/loading/loading_window.py

```diff
@@ -32,3 +28,6 @@ def increase_step(self, value):
     def closeEvent(self, event):
         self.filter_complete_signal.emit(1)
         event.accept()
+
+    def set_maximum(self, value):
```

Comment:
I guess the maximum value can always be 100

Reply:
The export methods in controller uses a different value, but for some reason he's not using this method to do that.
This can be seen [here](https://github.com/GraphFilter/GraphFilter/blob/112-parallel-graph-filtering/source/controller/controller.py#L188).